.github/workflows/spellcheck.yml

```diff
@@ -28,7 +28,6 @@ permissions:
 
 jobs:
   spellcheck:
-    name: Spellcheck (en_US)
```

Comment:
could you explain this deletion?

Reply:
https://github.com/instructlab/eval/pull/114#discussion_r1734773168